Adding the new Caption to the PreValue example (#4381)
umbraco/UmbracoDocs: Add-ons/UmbracoForms/Developer/Extending/Adding-a-Prevaluesourcetype.md

# Adding a Prevalue Source Type to Umbraco Forms

> The following example shows an illustrative custom prevalue source type that returns a hard-coded list of values. It can be extended for your needs via injection of services via the constructor. Dynamic Settings can be applied and validated [as shown in the other example custom types in this section](adding-a-type#validate-type-settings-with-validatesettings).

Reviewer: ```suggestion
The following example shows an illustrative custom prevalue source type that returns a hard-coded list of values. It can be extended for your needs via injection of services via the constructor. Dynamic settings can be applied and validated as shown in the [Validate type settings with ValidateSettings()](adding-a-type#validate-type-settings-with-validatesettings) article.
```

                new PreValue
                {
                    Id = 1,
                    Value = "item-one",

>                     Caption = "Item One

Reviewer 2: ```suggestion
                    Caption = "Item One"
```
